lib/spack/spack/test/cmd/load.py

```diff
@@ -96,6 +97,20 @@ def test_load_first(install_mockery, mock_fetch, mock_archive, mock_packages):
     load('--sh', '--first', 'libelf')
 
 
+def test_load_with_packages_yaml(install_mockery, mock_fetch,
```

Comment:
This should use the `config` fixture as well, so that it's more insulated from the users config.